rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandler.groovy

```diff
@@ -132,6 +132,11 @@ public class AWSBakeHandler extends CloudProviderBakeHandler {
   }
 
   @Override
+  String getBaseCommand() {
+    return awsBakeryDefaults.templateFile == "aws-chroot.json" ? "sudo" : ""
```

Comment:
Would it make more sense to add some config to rosco.yml identifying a list of templates that need to be run as `root`? If we hardcode just this template here, it make make it difficult to customize later.